### PR TITLE
File added

### PR DESCRIPTION
**Pull request by shubhamdalal49**

### CODE REVIEW

File: Shubham_Dalal/Bootstrap/ASSIGNMENT10/Shubham_assignment10.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                        </div>
+                        <p class="text-left" style="font-size: 14px;">
+                        Snehal Ghodmare posted a new material: Updated Document for HTML, CSS Content<br><span style="color: grey;">Dec 10</span></p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>     
+
+    </body>
```
Comment (Ashwin2812) [inline Shubham_Dalal/Bootstrap/ASSIGNMENT10/Shubham_assignment10.html]: Remove Extra Space

File: Shubham_Dalal/Bootstrap/ASSIGNMENT10/assignment10.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+h2{
+    color: white;
+    padding: 30px;
+    margin-left: 310px;
+}
+span{
+    margin-left: 330px;
+    font-size: 20px;
+    color: aliceblue;
+}
```
Comment (Ashwin2812) [inline Shubham_Dalal/Bootstrap/ASSIGNMENT10/assignment10.css]: Add extra line

**Review by Ashwin2812 — CHANGES_REQUESTED** [review]

File: Shubham_Dalal/Bootstrap/ASSIGNMENT10/Shubham_assignment10.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+       <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" />
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+        <link rel="stylesheet" href="assignment10.css" />
+        <title>Document</title>
```
Comment (nileshp1122) [inline Shubham_Dalal/Bootstrap/ASSIGNMENT10/Shubham_assignment10.html]: change title name

File: Shubham_Dalal/Bootstrap/ASSIGNMENT10/Shubham_assignment10.html [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+                        Snehal Ghodmare posted a new material: Bootstrap-Document<br><span style="color: grey;">Dec 16</span></p>
+                    </div>
+                </div>
+                <div class="card mt-4 pl-1 mr-2">
+                    <div class="card-body">
+                        <div class="rounded-circle bg-primary d-flex align-items-center" style="width: 40px; height: 40px">
+                            <img src="images.jpeg" alt="description" class="img-fluid mx-auto d-block" style="width: 30px; height: 30px">
+                        </div>
+                        <p class="text-left" style="font-size: 14px;">
+                        Snehal Ghodmare posted a new assignment: Evaluation 12Dec20<br><span style="color: grey;">Dec 12(Edited Dec 12)</span></p>
```
Comment (nileshp1122) [inline Shubham_Dalal/Bootstrap/ASSIGNMENT10/Shubham_assignment10.html]: check indentation on p tag

File: Shubham_Dalal/Bootstrap/ASSIGNMENT10/Shubham_assignment10.html [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+                        </li>
+                    </ul>
+                </div>
+            </div>
+            <div class="container col-8 float-right" style="width: 0px;">
+                <div class="card mt-4 pl-1 mr-2" style="box-shadow:2px 2px 4px grey;">
+                    <div class="card-body">
+                        <p class="text-left" style="color: grey; font-size: 16px;">
+                            <img src="images.jpeg" class="rounded-circle mr-1" alt="userimage" style="width: 30px; height: 30px;">
+                            Announce something to your class</p>
```
Comment (nileshp1122) [inline Shubham_Dalal/Bootstrap/ASSIGNMENT10/Shubham_assignment10.html]: check indentation on p tag

File: Shubham_Dalal/Bootstrap/ASSIGNMENT10/Shubham_assignment10.html [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+                    </div>
+                </div>
+                <div class="card mt-4 pl-1 mr-2">
+                    <div class="card-body">
+                        <div class="rounded-circle bg-primary d-flex align-items-center" style="width: 40px; height: 40px">
+                            <img src="images.jpeg" alt="description" class="img-fluid mx-auto d-block" style="width: 30px; height: 30px">
+                        </div>
+                        <p class="text-left" style="font-size: 14px;">
+                        Snehal Ghodmare posted a new material: Bootstrap-Document<br><span style="color: grey;">Dec 16</span></p>
```
Comment (nileshp1122) [inline Shubham_Dalal/Bootstrap/ASSIGNMENT10/Shubham_assignment10.html]: indentation on p

File: Shubham_Dalal/Bootstrap/ASSIGNMENT10/Shubham_assignment10.html [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+                        Snehal Ghodmare posted a new assignment: Evaluation 12Dec20<br><span style="color: grey;">Dec 12(Edited Dec 12)</span></p>
+                    </div>
+                </div>
+                <div class="card mt-4 pl-1 mr-2">
+                    <div class="card-body">
+                        <div class="rounded-circle bg-primary d-flex align-items-center" style="width: 40px; height: 40px">
+                            <img src="images.jpeg" alt="description" class="img-fluid mx-auto d-block" style="width: 30px; height: 30px; display: inline;">
+                        </div>
+                        <p class="text-left" style="font-size: 14px;">
+                        Snehal Ghodmare posted a new material: Updated Document for HTML, CSS Content<br><span style="color: grey;">Dec 10</span></p>
```
Comment (nileshp1122) [inline Shubham_Dalal/Bootstrap/ASSIGNMENT10/Shubham_assignment10.html]: indentation on p

File: Shubham_Dalal/Bootstrap/ASSIGNMENT10/Shubham_assignment10.html [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>     
+
```
Comment (nileshp1122) [inline Shubham_Dalal/Bootstrap/ASSIGNMENT10/Shubham_assignment10.html]: remove extra line or indentation

File: Shubham_Dalal/Bootstrap/ASSIGNMENT10/assignment10.css (hunk 2 above)
Comment (nileshp1122) [inline Shubham_Dalal/Bootstrap/ASSIGNMENT10/assignment10.css]: add extra line

**Review by nileshp1122 — CHANGES_REQUESTED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Review by Ashwin2812 — APPROVED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**